hfc/fabric/channel/channel.py

```diff
@@ -974,9 +974,9 @@ async def get_channel_config_with_orderer(self, tx_context, orderer):
             block = v.block
             break
 
-        block = BlockDecoder().decode(block.SerializeToString())
+        decoded_block = BlockDecoder().decode(block.SerializeToString())
```

Comment:
When decode==False, do we still needs to run the decode(...)?

Reply:
> @dongwangdw, I've created this PR because of #103, there is an issue with decoding of the config. So at least having an option fo fetch encoded config and decode it later with some other tool is helpful. Decoding was left there because someone might already use it, in addition function `get_channel_config` has an option to return encoded config so it would make sense for these two functions to have similar functionality.

OK. There is a data missing that you mentioned in #103. I would like to advise to investigate it deep for the root cause. And maybe a fix for it. It is very hard to fix it then let's have an easy workaround here. And your proposal would make more sense. Again I so much  appreciated your PR.

Reply:
in this case, it seems like we do, since after that it checks whether the correct block was fetched and fetches another one if needed.